iri-gen/index.ts

```diff
@@ -1,23 +1,60 @@
 import * as fs from 'fs';
+import { Pool, Client, PoolConfig } from 'pg';
 import { generateIRI } from './iri-gen';
+import 'dotenv/config'
 
-// Make sure we got a filename on the command line.
-if (process.argv.length < 3) {
-  console.log('You should provide the path to a JSON file');
-  process.exit(1);
+function setupPgPool() {
+  const poolConfig: PoolConfig = {
+    connectionString: process.env.DATABASE_URL || 'postgres://postgres:postgres@localhost:5432/regen_registry',
+  };
+
+  if (process.env.NODE_ENV === 'production') {
+    poolConfig.ssl = {
+      ca: fs.readFileSync(`${__dirname}/../config/rds-combined-ca-bundle.pem`),
+    };
+  }
+
+  const pool = new Pool(poolConfig);
+  return pool;
 }
 
-async function readFileAndGenerateIRI() {
-  try {
-    const path = process.argv[2];
+async function readFileAndGenerateIRI(path) {
     const rawdata = fs.readFileSync(path);
     const doc = JSON.parse(rawdata.toString());
-
     const iri = await generateIRI(doc);
-    if (iri) console.log(`The IRI for ${path} is: ${iri}`);
-  } catch (e) {
-    console.error(e);
+    return {doc, iri};
+}
+
+async function main() {
+  const parseArgs = require('minimist');
+  const argv = parseArgs(process.argv.slice(2), {'boolean': true});
+  // Make sure we got a filename on the command line.
+  if (argv._.length < 1) {
+    console.log('You should provide the path to a JSON file');
+    process.exit(1);
+  }
+  const insertFlag = argv.insert;
+  const path = argv._[0];
+  const {iri, doc} = await readFileAndGenerateIRI(path);
+  if (iri) {
+    console.log(`The IRI for ${path} is: ${iri}`)
+    if (insertFlag) {
+      const pool = setupPgPool();
+      try {
+        var client = await pool.connect();
```

Comment:
let's use ES6 everywhere
```suggestion
        const client = await pool.connect();
```